assets/constants.js

```diff
@@ -19,6 +19,8 @@ var constants = {
     "Mathematics", "Medicine", "Music", "Philosophy", "Physics", "Poetry",
     "Probability", "Programming", "Puzzles", "Reading", "Spanish", "Sport",
     "Statistics", "Trigonometry", "Welcome"],
+  "ACTIVITY_TYPE_EXPLORATION": "exploration",
```

Comment:
Note that the constants.js file is for constants that are shared between the frontend and the backend. If constants are added here they should be removed from feconf.py.